src/main/java/io/jenkins/pluginhealth/scoring/http/ProbeResultController.java

```diff
@@ -0,0 +1,54 @@
+package io.jenkins.pluginhealth.scoring.http;
+
+import javax.transaction.Transactional;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.jenkins.pluginhealth.scoring.model.ResultStatus;
+import io.jenkins.pluginhealth.scoring.service.PluginService;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping(path = "/results")
+public class ProbeResultController {
+
+    private final PluginService pluginService;
+
+    public ProbeResultController(PluginService pluginService) {
+        this.pluginService = pluginService;
+    }
+
+    @GetMapping(path = "")
+    public ModelAndView list() {
+        final ModelAndView modelAndView = new ModelAndView("probes/results");
+
+        modelAndView.addObject(
+            "probeResults",
+            getProbeResultsData()
+        );
+
+        return modelAndView;
+    }
+
+    @Transactional
+    public Map<String, Integer> getProbeResultsData() {
+        Map<String, Integer> probeResultsMap = new HashMap<>();
+
+        pluginService.streamAll().forEach(
+            plugin -> {
+                plugin.getDetails().values().forEach(probeResult -> {
+                    if (probeResult.status() == ResultStatus.SUCCESS) {
```

Comment:
While rendering the new view: for a given probe, I count its value towards its result only if its `status()`, for a given plugin, is a `SUCCESS`. And for each probe, there's a different definition for `SUCCESS`.

For example, as per the current code, for a given plugin, the `deprecation` probe is a `SUCCESS` if the **plugin is NOT deprecated**.  Due to this, the result of this probe in the new view is **1851**, meaning, 1851 plugins are not deprecated. And since this is opposite to what @jleon33 's [message](https://github.com/jenkins-infra/plugin-health-scoring/issues/96#issue-1424226722) captures (it says "32 are deprecated"), I have a question on this.

I guess I understand why we chose the current definition of `SUCCESS` for the `deprecation` probe, it's better to see the 1851 SUCCESS status than the 1851 FAILURE status in the DB.

**[Question]:** So to tackle this:
- should we change the current definition of `SUCCESS` for these kinds of probes? or
- should I write an `if-else` chain in place of this line, where I would match the `status()` of each probe with a custom `ResultStatus` value?

Reply:
> should we change the current definition of SUCCESS for these kinds of probes?

I'm really against this. It is a "bad" thing to see a plugin deprecated, so it's a failure.

Maybe the description of the probe needs to refer to the expected behavior of the probe more clearly.

Reply:
> I'm really against this. It is a "bad" thing to see a plugin deprecated, so it's a failure.

:+1: 

> Maybe the description of the probe needs to refer to the expected behavior of the probe more clearly.

I think that is a good idea, it makes it easier to know the success criteria for a probe. Because currently if I want to know the success criteria for a probe, I look at the code and look for the `message` String for `ProbeResult.success()` (which is not difficult to find as well).